app/test/test_server.py

```diff
@@ -11,31 +11,69 @@ def request():
 	request = MagicMock()
 	return request
 
+
 def test_home(request):
 	""" Do I render the home page? """
 	ret = server.home(request)
 	assert 'How much money are you about to spend?' in ret
 
+
 def test_invest(request):
 	""" Do I render the invest page? """
 	ret = server.invest(request, 2)
 	assert '$2 could be worth <strong>$16.23' in ret
 
+
 def test_initInvestSubmit(request):
 	""" Do I calculate the initial investment?  """
 	with patch('server.invest'):
 		server.initInvestSubmit(request)
 		server.invest.assert_called_once_with(request,1)
 
+
 def test_learnToInvest(request):
 	""" Do I render the resource page?"""
 	ret = server.learnToInvest(request)
 	assert 'Learn more about investing' in ret
 
-def test_saveOhlvc(request):
+
+def test_daily_stock(request):
+	""" Do I render the daily stock page? """
+	ret = server.daily_stock(request)
+	assert 'Provide a stock symbol' in ret
+	assert 'Daily Stock Data for' not in ret
+
+
+def test_daily_stock_with_symbol(request):
+	""" Do I render the daily stock page? """
+	request.args = {'symbol': ['INDEX_GSPC']}
+	ret = server.daily_stock(request)
+	assert 'Daily Stock Data for' in ret
+
+
+def test_get_ohlvc(request):
+	""" Do I return the recent stock data? """
+	pGetRecentOhlvc = patch.object(db, 'get_recent_ohlvc', return_value='stuff')
+	with pGetRecentOhlvc:
```

Comment:
what do you mean by 'context manager'?  i put in in a variable cause I thought it was too long and thought that 
`with patch.object(db, 'get_recent_ohlvc', return_value='stuff')` was ulgy and less readable.  I don't know which is better.  Consistency is best maybe? And I flip flop I think.